src/js/components/TaskDetailComponent.jsx

```diff
@@ -164,11 +167,17 @@ var TaskDetailComponent = React.createClass({
       );
     });
 
+    var ipAddressFields = this.getIpAddresses();
+    if (ipAddressFields != null && ipAddressFields.length > 0) {
+      ipAddressFields.unshift(<dt key="ip-addresses">IP Addresses</dt>)
+    }
```

Comment:
I did think about this, but the rest of the fields work by retrieving `<dd>` lists from another method and adds the `<dt>` in `render`. I wanted to stick with that convention.